lib/cisco_node_utils/vrf_af.rb

```diff
@@ -142,6 +182,21 @@ def default_route_target_export_evpn
     end
 
     # --------------------------
+    def route_target_export_stitching
+      cmds = config_get('vrf', 'route_target_export_stitching', @get_args)
+      cmds.nil? ? nil : cmds.sort
+    end
```

Comment:
It's not clear to me from the code here or from the tests below how this property is structured or how it is used. Is it a list of communities that are stitching-enabled? Can you provide some examples?

How does this property interact/coexist with the `route_target_export` property?